### PR TITLE
Preregistration

### PR DESCRIPTION
**Pull request by NMoksnes**
Hi Agnese! 
Here is the preregistration blog post.
Best,
Nandi

### CODE REVIEW

File: _posts/2022-03-12-preregistration_of_qualitative_studies.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+A preregistration makes the study less susceptible to bias (intentionally or unintentionally) as the analysis and expectations of the results can not be adjusted to fit the data. Furthermore, will a preregistration help to plan the study details and getting early input on the research.
+
+------------
+*This material is derived from the [CCG review of good enough practices][2], released under a [CC-BY 4.0 license][3].*
+
+[1]: <https://journal-buildingscities.org/articles/10.5334/bc.67/> "G. M. Huebner, M. J. Fell, and N. E. Watson, ‘Improving energy research practices: 1067 guidance for transparency, reproducibility and quality’, Buildings and Cities, vol. 2, 1068 no. 1, pp. 1–20, Jan. 2021, doi: 10.5334/bc.67." 
+
```
Comment (AgnesBelt) [inline _posts/2022-03-12-preregistration_of_qualitative_studies.md:19]: missing
```suggestion
[2]: https://zenodo.org/record/5911546#.YjsSSRDMI-R "Usher, William, Beltramo, Agnese, Gardumi, Francesco, Martin, Viktoria, & Petrarulo, Luca. (2022). CCG Platform - Body of Knowledge: Review of Good Practice (1.3). Zenodo. https://doi.org/10.5281/zenodo.5911546"
```
reference missing I think?

Comment (AgnesBelt) [reply _posts/2022-03-12-preregistration_of_qualitative_studies.md:19]: Also, missing reference [3]

File: _posts/2022-03-12-preregistration_of_qualitative_studies.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+layout: post
+title:  "Preregistration of qualitative studies"
+date:   2022-03-23
+author: Nandi Moksnes
+category: Practice
+tags: preregistration, qualtitiative studies
+---
+
+## Preregistration of qualitative studies
+When working on qualitative research a preregistration is promoted to ensure transparency, reproducibility and quality of the study [[1]]. A preregistration means that the researcher describes the research plan which will be undertake and analysis before starting the study. This includes basic information such as study aims, type of data collection, tools used in the study and analysis approach. The preregistration should be using standarized [forms, (e.g. any of these listed in Huebner et al. 2021)](https://journal-buildingscities.org/articles/10.5334/bc.67/#T1n) and registered *before* starting the research.
```
Comment (AgnesBelt) [inline _posts/2022-03-12-preregistration_of_qualitative_studies.md:11]:
```suggestion
When working on qualitative research a preregistration is promoted to ensure transparency, reproducibility and quality of the study [[1]]. A preregistration means that the researcher describes the research plan which will be undertake and analysis before starting the study. This includes basic information such as study aims, type of data collection, tools used in the study and analysis approach. The preregistration should be using standardised forms and registered *before* starting the research. Here some examples of generic preregistration forms that are applicable across different disciplines, as suggested by [Huebner et al. (2021)](https://journal-buildingscities.org/articles/10.5334/bc.67/#T1n):
- [OSF Prereg](https://osf.io/prereg/)
- [AsPredicted Preregistration](https://aspredicted.org/)
```

File: _posts/2022-03-12-preregistration_of_qualitative_studies.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+A preregistration makes the study less susceptible to bias (intentionally or unintentionally) as the analysis and expectations of the results can not be adjusted to fit the data. Furthermore, will a preregistration help to plan the study details and getting early input on the research.
```
Comment (AgnesBelt) [inline _posts/2022-03-12-preregistration_of_qualitative_studies.md:13]:
```suggestion
A preregistration makes the study less susceptible to bias (whether it is intentional or unintentional) as the analysis and expectations of the results can not be adjusted to fit the data. Furthermore, a preregistration helps to plan the study details and getting early input on the research, thus potentially enhancing the quality of the research conducted.
```

Comment (AgnesBelt) [reply _posts/2022-03-12-preregistration_of_qualitative_studies.md:13]: I feel like the first sentence here is not super clear.

File: _posts/2022-03-12-preregistration_of_qualitative_studies.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Preregistration of qualitative studies"
```
Comment (AgnesBelt) [inline _posts/2022-03-12-preregistration_of_qualitative_studies.md:3]: I think we could rename it "preregistration of research protocols"?
As it is not only valid for qualitative studies - although we thought of including this mainly to address requests from qualitative researchers that feel like the other guidelines are mainly quantitative research focused.

File: _posts/2022-03-12-preregistration_of_qualitative_studies.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+date:   2022-03-23
+author: Nandi Moksnes
+category: Practice
+tags: preregistration, qualtitiative studies
+---
+
+## Preregistration of qualitative studies
```
Comment (AgnesBelt) [inline _posts/2022-03-12-preregistration_of_qualitative_studies.md:10]:
```suggestion
## Preregistration of qualitative studies
```
I think this is not needed, the main header appears automatically from the post title. Correct @willu47?

**Review by AgnesBelt — CHANGES_REQUESTED** [review]
Hi @NMoksnes, Thanks a lot for this. Looks good. I've just made some small edits and added few comments here and there.

File: _posts/2022-03-12-preregistration_of_qualitative_studies.md (hunk 4 above)
Comment (NMoksnes) [reply _posts/2022-03-12-preregistration_of_qualitative_studies.md:3]: I think we can do either way, and you can decide which?

File: _posts/2022-03-12-preregistration_of_qualitative_studies.md (hunk 5 above)
Comment (willu47) [reply _posts/2022-03-12-preregistration_of_qualitative_studies.md:10]: Yep, correct.

**Review by willu47 — APPROVED** [review]

**Pull request merged**